ASP.NET Core sample/azure-pipelines/azure-pipelines.yaml

```diff
@@ -120,13 +119,20 @@ stages:
                 sudo mv saml2aws /usr/local/bin
                 
                 # Configure saml2aws
-                saml2aws configure --url=https://adfs.dfds.com/adfs/ls/IdpInitiatedSignOn.aspx --idp-provider=ADFS --mfa=Auto --session-duration=28800 --skip-prompt
+                # Configure saml2aws
+                if [[ $SAML2AWS_REGION == "\$(SAML2AWS_REGION)" ]]; then
+                  SAML2AWS_REGION=eu-west-1; export SAML2AWS_REGION
+                else
+                  SAML2AWS_REGION=$SAML2AWS_REGION; export SAML2AWS_REGION
```

Comment:
Why are you exporting again? It is already in the environment, right?

Reply:
When SAML2AWS_REGION is NOT defined in a variable or in a pipeline library, it will get the default value $(SAML2AWS_REGION). Not as an interpreted value, but actually $(SAML2AWS_REGION). God bless Azure DevOps for its stupidity.

So, if SAML2AWS_REGION is having no value, meaning it has $(SAML2AWS_REGION) as a text value, I am ensuring that it gets a real value instead.

The reason why eu-west-1 was chosen as default value is that it is the region where we have must of our cloud resources. If you have your stuff in another rehion, for instance eu-central-1, you can define that in  SAML2AWS_REGION environment variable in your pipeline library.

Reply:
That makes sense. I just don't think you need the `else` clause. Line 124 and 125. This will trigger when `SAML2AWS_REGION` has a value that is different from `$(SAML2AWS_REGION)`. And if that is the case, we dont need to do anything.